catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -74,7 +74,7 @@ public CatalogBackupPlugin() {
      */
     @Override
     public CreateResponse process(CreateResponse input) throws PluginExecutionException {
-        if (enableBackupPlugin) {
+        if (enableBackupPlugin && Requests.isLocal(input.getRequest())) {
```

Comment:
Post ingest plugin calls are only made by the framework which ensures that the input will not be null.